pkg/karmadactl/addons/descheduler/manifests.go

```diff
@@ -60,6 +60,10 @@ spec:
             initialDelaySeconds: 15
             periodSeconds: 15
             timeoutSeconds: 5
+          ports:
+            - containerPort: 10358
+              name: metrics
+              protocol: TCP
```

Comment:
Since the value `10358` is hard-coded here, I suggest adding `--secure-port` param to the `command` field as well, just like:

```yaml
- --bind-address=0.0.0.0
- --secure-port=10358
```

otherwise, supposing I defined `--secure-port` to other port or someone updated `defaultPort` in code, easily missed changes lead to inconsistency

maybe  other components ditto.

Reply:
because now other guys may don't known here 10358 is just the value of `--secure-port`